### PR TITLE
feat: implement intergrationRpc

### PR DESCRIPTION
**Pull request by junbeomlee**

### CODE REVIEW

**Review by Nipol — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -11,7 +11,14 @@
     "@oclif/command": "^1.5.13",
     "@oclif/config": "^1.12.12",
     "@oclif/plugin-help": "^2.1.6",
-    "tslib": "^1.9.3"
+    "@types/faker": "^4.1.4",
```
Comment (Nipol) [inline package.json]: faker should be a move to devDependencies.

Comment (junbeomlee) [reply package.json]: fixed it!

Comment (Nipol) [reply package.json]: Good!

File: src/types/Integration.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import "reflect-metadata";
+
+export class Integration {
+  integrationId: string;
+  userId: string;
```
Comment (zero1ne-dev) [inline src/types/Integration.ts]: The userId will probably be a number.

Comment (junbeomlee) [reply src/types/Integration.ts]: thx i fixed it

**Pull request merged**